src/index.js

```diff
@@ -1,7 +1,9 @@
 import {UserChip} from './UserChip/UserChip.jsx';
 import {Toolbar} from './Toolbar/Toolbar.jsx';
+import {isVisibleComponent} from './VisibleComponent/VisibleComponent.jsx';
```

Comment:
No named import needed. After rebasing all the other imports should be unnamed aswell.

Reply:
Nope, `isVisibleComponent` is a named export (and should stay as it is as it is a method).